Packages/com.unity.inputsystem/InputSystem/Plugins/DualShock/DualShockGamepad.cs

```diff
@@ -118,6 +124,9 @@ protected override void FinishSetup()
             R2 = rightTrigger;
             L3 = leftStickButton;
             R3 = rightStickButton;
+
+            if (m_Description.capabilities != null && m_Description.interfaceName == "HID")
+                hidDescriptor = HID.HID.HIDDeviceDescriptor.FromJson(m_Description.capabilities);
```

Comment:
I decided to put this into the "base" DualShock class instead of copy-pasting it amongst the various DualShock/DualSense HID classes. I think ideally we'd have a DualShockHID common class that all the variants inherit from.